website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -104,54 +104,57 @@ models:
 
 <VersionBlock firstVersion="1.6">
 
-## Materialized view
+## Materialized Views
 
-The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html) and refreshes them for every subsequent `dbt run` that you execute. For more information, see [Refresh Materialized Views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) in the Redshift docs.
+<Changelog>
 
-Materialized views support the optional configuration `on_configuration_change` with the following values: 
-- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. 
-- `continue` &mdash; allows runs to continue while also providing a warning that the model was not executed
-- `fail` &mdash; forces runs to fail if a change is detected in a materialized view
+  - **v1.6.0:** Introduced support for `materialized_view`
 
-Additionally, you can apply the `auto_refresh` configuration to have Redshift [automatically refresh](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) the materialized view for you. This action can applied without the need to rebuild the materialized view. 
+</Changelog>
 
-You can create a materialized view by editing _one_ of these files:
-- the SQL file for your model
-- the `dbt_project.yml` configuration file
+The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html).
+Redshift-specific configuration includes the typical `dist`, `sort_type`, `sort`, and `backup`.
+For materialized views, there is also the `auto_refresh` setting, which allows Redshift to [automatically refresh](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) the materialized view for you.
+The remaining configuration follows the general [materialized view](/docs/build/materializations#Materialized-View) configuration.
+There are also some limitations that we hope to address in the next version.
 
-The following examples create a materialized view: 
+### Monitored Configuration Changes
 
-<File name='models/YOUR_MODEL_NAME.sql'>
+The settings below are monitored for changes applicable to `on_configuration_change`.
 
-```sql
-{{
-  config(
-    materialized = 'materialized_view',
-    auto_refresh = False,
-    on_configuration_change = 'apply',
-  )
-}}
-```
+#### Dist
 
-</File>
+Changes to `dist` will result in a full refresh. Redshift requires a materialized view to be
```

Comment:
Is this at next `dbt run`?

Reply:
Most likely. Though if the materialized view was manually created and then dbt run was run for the first time with that materialized view as a dbt model, it would initiate a full refresh there too. It's depends on whether the materialized view already exists and if the dist config in the dbt model is different from that of the materialized view in the database, not if this is the first or second time dbt is run. That sounds like semantics, but that's my answer. Regardless, if you think this is less confusing to our users by inserting that, I'll definitely make the change.

Reply:
Ah okay - that's fine. The issue is `refresh` takes place in multiple flows.  How does this sound?

```suggestion
Changes to `dist` will result in a full refresh of the existing materialized view (at the time of the next `dbt run` of the model). Redshift requires a materialized view to be
```

Reply:
> The issue is refresh takes place in multiple flows. How does this sound?

That's correct. It will happen in three scenarios:

- The user has requested it via `--full-refresh`
- There is an existing object that is not a materialized view
- There is an existing materialized view with different configuration, such that some piece of that different configuration requires a full refresh, and the user has supplied `on_configuration_change='apply'` (this is also the default, so omitting this setting behaves the same for this scenario)